pages/api/register.ts

```diff
@@ -0,0 +1,39 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { PrismaClient } from '@prisma/client'
+
+const prisma = new PrismaClient()
+
+type Data = {
+  message: string
+}
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<Data>
+) {
+  if (req.method === 'POST') {
+    interface BhaktData {
+      full_name: string,
+      password_hash: string,
+      email: string
+    }
+
+    const full_name = req.body.full_name.trim().length > 0 ? req.body.full_name.trim() : undefined;
+    const password_hash = req.body.password.trim()
+    const email = req.body.email.trim().length > 12 ? req.body.email.trim().toLowerCase() : undefined;
```

Comment:
@dewa-himanshu24 Why email is undefined ?

Reply:
@jainendra What to send incase email is not following criteria.